### ISSUE TITLE
MS 20: Thumbnails not displaying

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#265]
### Descriptive Summary

@tmariamora noticed that morning that some of the thumbnails within the RMB-PJ collection are not displaying. Example:

https://digitalcollections.library.ucsc.edu/catalog?locale=en&q=Zoo&search_field=title

<img width="1243" alt="Screen Shot 2020-05-05 at 8 46 00 AM" src="https://user-images.githubusercontent.com/32003407/81088383-b8daf600-8eaf-11ea-944c-062b0d2d7669.png">

Error message is appearing when you click on one of the images with a missing thumbnail: https://digitalcollections.library.ucsc.edu/concern/works/zc77ss65z?locale=en

**Comment by NedHenry** [comment]
I believe these only appear to logged in users (i.e. administrators). These are complex objects that failed to completely ingest due to failures during the last ingest (the complex object ingest errors that it took me so long to fix). Now Fedora has problems deleting these works. Because they don't show up to the public, it hasn't been a top priority yet to figure out why Fedora doesn't want to delete them and fix it.

**Comment by rmjaffe** [comment]
**Update** 
Ned's comments from Slack: 
Yeah, as I suspected. You get this error when you click on those works you saw that have no thumbnail. Those are works that failed in a funny way during the previous ingest (before I figured out some of the complex object issues). Now we have re-ingested the works, so these are failed duplicates that can and should be deleted. The only problem is that Fedora is having an issue fully deleting them. It should be possible to sort this issue out, but for now I’ve just made those works private and put it off to do other stuff (I really needed a break from complex object ingest problems at the time).
Fortunately, only admins can get to that URL or view those works. If you aren’t already logged in, you don’t see those failed works in search results and the link you posted goes to a login screen.
In any case, thanks for the report!
